src/main/java/uk/gov/hmcts/befta/util/ExpectedValuePlaceholder.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.befta.util;
+
+import java.text.NumberFormat;
+import java.text.SimpleDateFormat;
+
+public enum ExpectedValuePlaceholder {
+
+    ANYTHING_PRESENT(false),
+    ANYTHING_IF_EXISTS(true),
+
+    ANY_NULLABLE(true),
+    ANY_STRING_NULLABLE(true),
+    ANY_NUMBER_NULLABLE(true),
+    ANY_INTEGER_NULLABLE(true),
+    ANY_FLOATING_NULLABLE(true),
+    ANY_DATE_NULLABLE(true),
+    ANY_TIMESTAMP_NULLABLE(true),
+    ANY_OBJECT_NULLABLE(true),
+
+    ANY_NOT_NULLABLE(false),
+    ANY_STRING_NOT_NULLABLE(false),
+    ANY_NUMBER_NOT_NULLABLE(false),
+    ANY_INTEGER_NOT_NULLABLE(false),
+    ANY_FLOATING_NOT_NULLABLE(false),
+    ANY_DATE_NOT_NULLABLE(false),
+    ANY_TIMESTAMP_NOT_NULLABLE(false),
+    ANY_OBJECT_NOT_NULLABLE(false);
+
+
+    private boolean nullable;
+
+    ExpectedValuePlaceholder(boolean nullable) {
+        this.nullable = nullable;
+    }
+
+    public boolean isNullable() {
+        return this.nullable;
+    }
+
+    public boolean isValidFormat(Object actualObject) {
```

Comment:
could we rename this so accepts(Object actualValue)
to read a code like if(somePlaceholder.accepts(actualValue))

Reply:
Otherwise nice one.

Reply:
Changed to accepts(Object actualValue)